src/test/lambda/commands/copyLambdaUrl.test.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { createStubInstance, SinonStubbedInstance, stub, SinonStub, spy, SinonSpy } from 'sinon'
+import { copyLambdaUrl, noLambdaFuncMessage } from '../../../lambda/commands/copyLambdaUrl'
+import { LambdaFunctionNode } from '../../../lambda/explorer/lambdaFunctionNode'
+import { DefaultLambdaClient, LambdaClient } from '../../../shared/clients/lambdaClient'
+import globals from '../../../shared/extensionGlobals'
+import { addCodiconToString } from '../../../shared/utilities/textUtilities'
+import { buildFunctionUrlConfig } from '../../shared/clients/defaultLambdaClient.test'
+
+describe('copy lambda function URL to clipboard', async () => {
+    let client: SinonStubbedInstance<LambdaClient>
+    let node: Pick<LambdaFunctionNode, 'name' | 'regionCode'>
+    let clipboardFunc: SinonStub
+    let quickPickUrlFunc: SinonStub
+
+    const nodeName = 'my-node-name'
+
+    beforeEach(async () => {
+        client = createStubInstance(DefaultLambdaClient)
+        clipboardFunc = stub()
+        quickPickUrlFunc = stub()
+
+        node = { name: nodeName, regionCode: 'myRegion' }
+    })
+
+    it('Single URL exists', async () => {
+        const urlConfig = buildFunctionUrlConfig({ FunctionUrl: 'url1', FunctionArn: 'arn1' })
+        client.getFunctionUrlConfigs.resolves([urlConfig])
+
+        await copyLambdaUrl(node, client, clipboardFunc)
+
+        assert(client.getFunctionUrlConfigs.calledOnceWith(nodeName))
+        assert(clipboardFunc.calledOnceWith(urlConfig.FunctionUrl, 'URL'))
```

Comment:
should be able to actually check the clipboard here. Then we don't need these 2 spies.